### ISSUE TITLE
esp32-camera: Downloading  a file from ftp server to SD card? 

### ISSUE THREAD

**Issue by Yuxekova** [JohnnyB1290/ESP32-FTP-Client#4]
Hi, I'm using esp32-cam with SD card in it. I want to transfer a .mp4 file which size probably between 1 or 2 MB. Can i upload files more than ram size to SD card?

**Comment by dizcza** [comment]
@af-unlu yes as it allocates a static buffer of size `FTP_CLIENT_RESPONSE_BUFFER_SIZE` (1024 bytes by default) to transfer data in and out.
